runbooks/mcp-to-permissioned-fps.md

```diff
@@ -244,7 +244,7 @@ We recommend providing the deployer account with 1 ETH on Mainnet and 10 ETH on
         -v ./deploy_config:/deploy_config.json \
         -v ./deployments.json:/deployments.json \
         -v ./outputs:/outputs \
-        kfoplabs/upgrade-v1.3.0-v1.6.0-permissioned:latest \
+        kfoplabs/upgrade-v1.3.0-v1.6.0-permissioned@sha256:dfb661ac34e997694eb0aaa2536515992cc0a3d663bfe3adbac0deb7aa1dd159 \
```

Comment:
What is this a hash of?

Reply:
the version of the docker image Kelvin used to test the upgrade runbook

Reply:
https://github.com/ethereum-optimism/superchain-ops/pull/345#discussion_r1802063875

Reply:
Is there a way to verify that's the right docker image?

Reply:
I'll follow up here when I hear back from Kelvin on slack

Reply:
This docker image was manually build from the commit from the [op-contracts/v1.6.0 release](https://github.com/ethereum-optimism/optimism/tree/perm/op-contracts/v1.6.0). It wasn't added to the CI because its an old commit.